app/views/provider/invitee_signups/show.html.erb

```diff
@@ -1,6 +1,16 @@
 <%= javascript_pack_tag 'SignUpPage' %>
 
-<div id="old-signup-page-wrapper">
+<%= tag("div", :id => "login-page-container", :"data-signup-props" => {:path => provider_invitee_signup_path(@invitation.token),
```

Comment:
is it really a login page? by the look of the properties seems to be a signup

Reply:
@josemigallas the change in the name is because of this